Sources/FronteggSwift/embedded/FronteggWKContentController.swift

```diff
@@ -72,6 +72,22 @@ class FronteggWKContentController: NSObject, WKScriptMessageHandler {
             FronteggAuth.shared.loginWithSSO(email: message.payload)
         case "loginWithSocialLogin":
             FronteggAuth.shared.loginWithSocialLogin(socialLoginUrl: message.payload)
+        case "loginWithPassword":
+            guard let data = try? JSONSerialization.jsonObject(with: Data(payload.utf8), options: []) as? [String: String],
+                let email = data["email"],
+                let password = data["password"] else {
+                    print("Invalid payload for loginWithPassword")
+            
+                    return
+                }
+
+            FronteggAuth.shared.saveWebCredentials(domain: "autheu.davidantoon.me", email: email, password: password) { success, error in
+                if success {
+                    print("Credentials saved successfully for \(email)")
```

Comment:
we need to check how we can hold the screen until the user accept/reject the save password